### PR TITLE
Add smartBCH contracts, configure price service to fetch prices

### PR DESCRIPTION
**Pull request by mainnet-pat**
- Add smartbch deployments
- Support fetching bch price
- Add price service configuration for smartbch
- Add mistswap price oracle, change bch usd price ticker on coingecko to wrapped-bch
- smartBCH deployments using safe singleton factory

### What was wrong?

N/A

Closes #

N/A

### How was it fixed?

Added new safe deployment info from smartBCH

@gnosis/safe-services

### CODE REVIEW

**Comment by mainnet-pat** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Uxio0** [comment]
Related to https://github.com/safe-global/safe-deployments/pull/90

File: safe_transaction_service/tokens/services/price_service.py [hunk 1]
```diff
@@ -46,6 +47,14 @@
 )
 from ..tasks import EthValueWithTimestamp, calculate_token_eth_price_task
 
+
+class MistswapOracle(UniswapV2Oracle):
```
Comment (Uxio0) [inline safe_transaction_service/tokens/services/price_service.py:51]: Oracles should go in https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/oracles/oracles.py

**Review by Uxio0 — CHANGES_REQUESTED** [review]

**Comment by mainnet-pat** [comment]
Hey, yes, sorry for not having time to finish it. I'd need to update the oracle repository first.

**Pull request closed without merging**